docs/source/examples/de_aps_data_management.md

```diff
@@ -0,0 +1,274 @@
+# Integration with APS Data Management (DM)
+
+> The APS Data Management System is a system for gathering together experimental
+> data, metadata about the experiment and providing users access to the data
+> based on a users role.
+
+APS beamlines have been using DM to process acquired data (with DM workflows).
+See the [documentation](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/HowTos/Getting-Started) for more details.  This document describes a way to integrate DM with the Bluesky framework.  Before the integration is described, a brief overview of DM...
+
+## DM Overview
+
+For integration with the Bluesky framework, we provide a minimal explanation for these parts of DM:
+
+- experiment
+- DAQ
+- workflow
+- upload
+
+### experiment
+
+> The experiment is the central object for creating entries in the Data
+> Management System.
+
+Typically, a DM experiment will manage all data for a single proposal/ESAF at a
+beamline.  Beamline staff will create the DM experiments they will need.
+
+The **name** of the experiment will be used by bluesky to coordinate acquired data and runs with DM.
+
+An experiment will create data storage with permissions for the beamline data
+acquisition account and all the users listed in the proposal.
+
+More info [here](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/HowTos/Getting-Started#getting-some-data-into-the-system)
+
+### DAQ
+
+> ... overall purpose of this system is to get data into the system and provide
+> access control to that data.
+
+In DM, a *DAQ* monitors a local file directory tree (on a specific workstation)
+and copies any new content to the DM experiment's data storage.
+
+More info [here](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/HowTos/Getting-Started#getting-files-into-data-management)
+
+### workflow
+
+> A workflow is simply a set of defined steps that will process data.
+
+The steps to process data: move acquired data to
+computation hosts (local or remote), run the computation applications, then move
+any results to a final destination.  A workflow is capable of other activities,
+such as communicating back to EPICS.
+
+More info [here](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/HowTos/Getting-Started#workflows-and-processing-data)
+
+### upload
+
+In addition to automatic file copying with a DAQ, DM provides for single file
```

Comment:
Just to clarify, you can "upload" the whole directories as well. The difference between the "file" and "directory" processing modes is that in the file mode we files are processed (transfer, catalog, invoke workflow, etc) one by one, while  in the directory mode there will be a single transfer of the whole directory, after which all files will be cataloged, etc. The advantage of the file mode is that you can keep track of progress and see/recover from errors easier, while the advantage of the directory mode is that it is more efficient, especially for large number of small files, where any new transfer connection is more expensive relative to the data transfer time.

Reply:
Thanks!